specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/virtualMachineImage.json

```diff
@@ -789,6 +789,90 @@
       }
     }
   },
+  "x-ms-paths":{
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/publishers/{publisherName}/artifacttypes/vmimage/offers/{offer}/skus/{skus}/versions?$expand=Properties": 
```

Comment:
Would this x-ms-path will work for showing "imageDeprecationStatus" property as well? 
Because in the REST-API we have to give this url to get the imageDeprecationStatus:
"/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/publishers/{publisherName}/artifacttypes/vmimage/offers/{offer}/skus/{skus}/versions?$expand=Properties/**imageDeprecationStatus**"

Reply:
Yes it will show as all properties will be mapped to the return type